src/Microsoft.AspNet.Authentication/ClaimsTransformationAuthenticationHandler.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
User code is pretty straight forward `services.ConfigureClaimsTransformation(Func<ClaimsPrincipal, ClaimsPrincipal>)`  

The xform middleware goes in the pipeline after all of the other auth middlewares.  Unfortunately understanding middlewares is pretty much required in ASPNET5 now.